plugins/notifications-backend/src/service/router.ts

```diff
@@ -177,9 +179,31 @@ export async function createRouter(
     return users;
   };
 
+  const filterProcessors = (payload: NotificationPayload) => {
+    const result: NotificationProcessor[] = [];
+
+    for (const processor of processors) {
+      if (processor.getNotificationFilters) {
+        const filters = processor.getNotificationFilters();
+        if (filters.minSeverity && payload.severity) {
+          if (
+            notificationSeverities.indexOf(payload.severity) >
```

Comment:
What happens if the payload does not have severity? The default `normal` is set after processing the options.

Reply:
in that case the processor is included (not filtered out)

Reply:
I mean that the `payload.severity` can be undefined at this phase

Reply:
added default